examples/custom_code_deployment/steps/prediction_service_loader/prediction_service_loader.py

```diff
@@ -0,0 +1,102 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+from typing import cast
+
+from zenml.integrations.kserve.model_deployers import KServeModelDeployer
+from zenml.integrations.kserve.services import KServeDeploymentService
+from zenml.integrations.seldon.model_deployers.seldon_model_deployer import (
+    SeldonModelDeployer,
+)
+from zenml.integrations.seldon.services.seldon_deployment import (
+    SeldonDeploymentService,
+)
+from zenml.steps import BaseStepConfig, step
```

Comment:
Seems to me like this would import both the Kserve as well as the Seldon specific integrations which would fail if not both are installed. how would you feel about moving the imports into the respective functions. 

In general one should be able to run the example wiuth either seldon or kserve installed.

Reply:
As the two functions are very similar they could maybe even be unified and use the active_stack to get the active model deployer, in that case you might not even need explicit imports?

Reply:
Yes, that's true, users would be required to have both installed! it should be either changed to be imported after getting the model deployer option in the command line or would be even better to get it using the active stack.

One question, would it be better to just split the examples or create multiple sub-examples in the custom code example? cause there will be a TensorFlow (maybe sklearn) example added?